plugins/common/tls/config_test.go

```diff
@@ -42,6 +42,34 @@ func TestClientConfig(t *testing.T) {
 				TLSKeyPwd: "",
 			},
 		},
+		{
+			name: "fail with enc tls key",
+			client: tls.ClientConfig{
+				TLSCA:     pki.CACertPath(),
+				TLSCert:   pki.ClientCertPath(),
+				TLSKey:    pki.ClientEncKeyPath(),
+				TLSKeyPwd: "changeme",
+			},
+			expNil: true,
+			expErr: true,
```

Comment:
testutil/pki/tls-certs.sh line 82 sets the password to "changeme". This test has the same password but is expected to be nil and expected to have an error. This seems backward to me. Shouldn't this pass?